imap_processing/idex/idex_l1a.py

```diff
@@ -127,6 +76,7 @@ def __init__(self, packet_file: Union[str, Path], data_version: str) -> None:
 
         Notes
         -----
+            # TODO should this be assumed?
```

Comment:
I think so. One L0 can contain multiple packets but should produce one l1a file per APID. And in this case, this class is processing science packets, it will result in one file.

Reply:
OK great